docs/reference-guides/theme-json-reference/theme-json-living.md

```diff
@@ -1,16 +1,27 @@
-# Version 2 (living reference)
+# Theme.json Version 2 (living reference)
```

Comment:
I don't think we need "(living reference)" here. It feels redundant. At the very least we could update to just "Reference" to match the title of the other doc.

Reply:
Agreed 👍